common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -474,16 +488,13 @@ def test_user_already_enrolled(self, mock_get_course_runs):
 
         assert response.status_code == 201
         assert CourseEnrollment.is_enrolled(self.user, self.course.id)
-
-        course_entitlement.refresh_from_db()
-        assert CourseEnrollment.is_enrolled(self.user, self.course.id)
```

Comment:
is there a reason we are taking out this assertion?

Reply:
It is a duplicate of the one above it.